### ISSUE TITLE
'Carousel' cannot be used as a JSX component

### ISSUE THREAD

**Issue by pyDjangoDev** [YIZHUANG/react-multi-carousel#346]
![2022-05-31_03-21](https://user-images.githubusercontent.com/61251512/171050095-2b7af2dc-fe88-449e-8a07-346b652c88f4.png)
![2022-05-31_03-13](https://user-images.githubusercontent.com/61251512/171050110-f3a9b117-ac1e-4f0c-8517-fa466ba51b13.png)

Could someone help me, please?

react-multi-carousel@^2.8.0

Thanks in advance.

**Comment by vladimirmyshkovski** [comment]
Same error with the same version.

**Comment by vladimirmyshkovski** [comment]
@pyDjangoDev remove `node_modules` and `yarn.lock` or `package-lock.json` file and reinstall dependencies. In my case it's solve the problem.

**Comment by pyDjangoDev** [comment]
I just tried to do it in that way. but I still get the same error.
Anyway, thanks @vladimirmyshkovski

**Comment by shaffi-ahuja** [comment]
@pyDjangoDev Were you able to resolve this issue... I am also facing the same

**Comment by Marthijs-Berfelo** [comment]
@pyDjangoDev I encountered the same issue with an older version however only when running a different NODE version. 

- using latest LTS: `v16.16.0` the problem occurred
- using older version: `v16.10.0` no problem

**Comment by MuizU** [comment]
> @pyDjangoDev @shaffi-ahuja I encountered the same issue with an older version however only when running a different NODE version.
> 
> * using latest LTS: `v16.16.0` the problem occurred
> * using older version: `v16.10.0` no problem
> 
> **UPDATE**
> 
> even NODE versions `v16.15.0` and `v16.15.1` work for me.

This didn't work for me despite using all three versions.

**Comment by rortan134** [comment]
Try adding `<></>` (React.Fragment) wrapping the `{}` as a workaround like this:
```
<Carousel >
  <>
     {avatars.map((_) =>  /*...*/ )}
  </>
</Carousel />
```
See more: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58286#issuecomment-567690424

**Comment by pyDjangoDev** [comment]
> Try adding `<></>` (React.Fragment) wrapping the `{}` as a workaround like this:
> 
> ```
> <Carousel >
>   <>
>      {avatars.map((_) =>  /*...*/ )}
>   </>
> </Carousel />
> ```
> 
> See more: [DefinitelyTyped/DefinitelyTyped#18051 (comment)](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/18051#issuecomment-567690424)

**Comment by pyDjangoDev** [comment]
> @pyDjangoDev Were you able to resolve this issue... I am also facing the same

I didn't resolve it at that moment. so I used the react-slick, an alternative.

**Comment by MuizU** [comment]
It worked for me after I upgraded types/react to 18.0.1

**Comment by eucciferri** [comment]
In my case I was able to solve this problem by updating the dependencies of the packages, executing the command:
```yarn upgrade```

**Comment by zhenglib** [comment]
I had same issue, and I solved it with node v 16.13.0

1. Remove node_modules
2. nvm use 16.13.0
3. npm i --save
4. npm start

This solved my issue.
